### PR TITLE
Some proposed changes to the web automation section

### PR DESCRIPTION
**Pull request by israelconesa**
Hi,

I'm a big fan of your work so this is my humble attempt to help out with some ideas.

Thanks,

Israel.

### CODE REVIEW

**Comment by angelovstanton** [comment]
Hi Israel!

Thank you for the contribution and the help! I will give my best to review the fixes probably next week or I will ask a colleague to help with this.

All the best,
Anton

File: docs/general-information/general-information.md [hunk 1]
```diff
@@ -12,12 +12,12 @@ anchors:
...
 Overview
 --------
-BELLATRIX is not a single thing it contains multiple framework libraries, extensions and tools. The tool is built to be cross-platform.
+BELLATRIX is not a single thing, it contains multiple framework libraries, extensions and tools. The tool is built to be cross-platform.
 
 Simple Installation
 ------------------
-1. Download the BELLATRIX
-2. Unzip it. Open BellatrixTestFramework.sln in Visual Studio 
+1. Download BELLATRIX.
```
Comment (ViktorVakareev) [inline docs/general-information/general-information.md]: Download the BELLATRIX **file/zip**.

Comment (israelconesa) [reply docs/general-information/general-information.md]: I'll wait for all the feedback before making changes.

Comment (israelconesa) [reply docs/general-information/general-information.md]: Change made

**Review by angelovstanton — APPROVED** [review]

**Pull request merged**